std_msgs/Header header
float64 cam_distance_reader          # cam_distance_reader reading
float64 cam_distance_reader_var      # cam_distance_reader_var reading

